src/main/java/org/galatea/starter/service/IexHistoricalClient.java

```diff
@@ -0,0 +1,28 @@
+package org.galatea.starter.service;
+
+import java.util.List;
+import org.galatea.starter.domain.IexHistoricalPrices;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+/**
+ * A Feign Declarative REST Client to access endpoints from the Free and Open IEX API to get market
+ * data. See https://iextrading.com/developer/docs/
```

Comment:
this javadoc is copy/pasted, not applicable to this client?

Reply:
Yes. I copied it. Will make changes.